File: SofiForce.Web/ClientApp/src/app/core/Models/StatisticalModels/RepresentativeJourneyKBIModel.ts
export interface RepresentativeJourneyKBIModel {
    allClients: number;
    clients: number;
    allRepresentatives: number;
    representatives: number;
    clientPerformance: number;
    representativePerformance: number;
    perormancePercentage: number;
    perormanceLabel: string;
    detailModels: RepresentativeJourneyKBIDetailModel[];
}

export interface RepresentativeJourneyKBIDetailModel {
    branchId: number;
    branchCode: string;
    branchName: string;
    allClients: number;
    clients: number;
    allRepresentatives: number;
    representatives: number;
    clientPerformance: number;
    representativePerformance: number;
    perormancePercentage: number;
    perormanceLabel: string;
}

File: SofiForce.Models/Models/StatisticalModels/RepresentativeJourneyKBIModel.cs
using System;
using System.Collections.Generic;

namespace SofiForce.Models.StatisticalModels
{
    public class RepresentativeJourneyKBIModel
    {

        public int AllClients { get; set; }
        public int Clients { get; set; }
        public int AllRepresentatives { get; set; }
        public int Representatives { get; set; }
        public decimal ClientPerformance { get; set; }
        public decimal RepresentativePerformance { get; set; }
        public decimal PerormancePercentage { get; set; }

        public string PerormanceLabel { get; set; }


        public IEnumerable<RepresentativeJourneyKBIDetailModel> DetailModels { get; set; } = new List<RepresentativeJourneyKBIDetailModel>();

    }

    public class RepresentativeJourneyKBIDetailModel
    {

        public int BranchId { get; set; }
        public string BranchCode { get; set; }
        public string BranchName { get; set; }
        public int AllClients { get; set; }
        public int Clients { get; set; }
        public int AllRepresentatives { get; set; }
        public int Representatives { get; set; }
        public decimal ClientPerformance { get; set; }
        public decimal RepresentativePerformance { get; set; }
        public decimal PerormancePercentage { get; set; }

        public string PerormanceLabel { get; set; }

    }
}
